mewarpx/mewarpx/diags_store/field_diagnostic.py

```diff
@@ -127,7 +130,7 @@ def do_post_processing(self):
                     )
 
                     for parameter in self.diag_data_list:
-                        plot_name = parameter + "_" + datafolder.replace("diags/diags", "")
+                        plot_name = self.write_dir + "/" + parameter + "_" + datafolder.replace(self.write_dir + "/" + self.name, "")
```

Comment:
Using `os.path.join` would be cleaner than doing `self.write_dir + "/" + parameter + ...`